djangocms_bootstrap4/contrib/bootstrap4_grid/models.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+from functools import partial
+
+from django.db import models
+from django.conf import settings
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext, ungettext, ugettext_lazy as _
+from django.utils.safestring import mark_safe
+
+from cms.models import CMSPlugin
+
+from ...constants import DEVICE_SIZES
+from ...utils import TagTypeField, AttributesField, IntegerRangeField
```

Comment:
move all these fields to a `fields.py` file

Reply:
I don't understand, they are in the core djangocms_bootstrap4.utils. Should I use fields.py instead. If yes can you explain to me a bit the hirarchy of:
- fields.py
- helpers.py
- utils.py
and what to put where when?